.version

```diff
@@ -0,0 +1 @@
+5.2.0-SNAPSHOT
```

Comment:
This technique was used in iOS and I thought it would be good to port to all our SDKs. Basically, a `.version` file in the root folder will always contain the current version. We can use that to substitute this string on all the files that contain the old version number with the new one. Since each platform has its own versioning system, this is a way to abstract that. Lmk if you have any concerns.

Reply:
I agree 💯